backend/controllers/LocationsController.ts

```diff
@@ -91,6 +92,27 @@ const LocationsController = {
       return res.status(500).json({ error: err.message });
     }
   },
-};
+
+  async addLocationComments(req: Request, res: Response) {
+    try {
+      const {id,comment}=req.body;
+      const locations = await Location.findById(id);
+      if (!locations||!locations.comments) {
+        return res.status(400).json({ error: "Location or comments doesn't exist" });
+      }
+      else{locations.comments.push(comment)};
+      const updateLocations=await Location.findByIdAndUpdate(id,{comments:locations.comments}, { 'new': true })
```

Comment:
'else' is redundant
because in case there is no location or comment in it, backend return error and this code won't be reached